src/http_server.rs

```diff
@@ -30,9 +68,19 @@ impl<T: Controller> HttpServer<T> {
 
         let addrs: Vec<_> = self.controller.http_as_addrs().unwrap().collect();
 
-        thread::Builder::new().name("HttpServer".to_owned())
+        debug!("Starting server with cert: {:?} and key: {:?}", cert, key);
+
+        let mut config = TlsSniConfig::new();
+        config.add_ssl_cert("foxbox.local".to_owned(), cert, key).unwrap();
+        config.add_ssl_cert("test.knilxof.org".to_owned(), self.controller.get_remote_tls_crt(), self.controller.get_remote_tls_key()).unwrap();
```

Comment:
On the local network you want to expose port 443, yes. But to the tunnel client, it would make more sense to me to expose either a high port or a unix socket (also depending on how we implement the tunnel client).

But yes, if we ever want more than one local and one bridged domain name, then you're right that SNI is useful. Could be the case if we ever want to activate https://github.com/fxbox/RFC/pull/4 and https://github.com/fxbox/RFC/pull/5 simultaneously.

Doesn't really matter though, I was just curious. :)